### ISSUE TITLE
Stuck in Activities Overview

### ISSUE THREAD

**Issue by benscruton** [RensAlthuis/vertical-overview#69]
On Fedora 34 with Gnome version 40.4.0, after running `sudo dnf upgrade` this morning I got stuck in the Activities Overview and couldn't get out of it.  I was able to move around within the activities overview -- switching between workspaces worked fine, and I was also able to switch back and forth between the Activities Overview and the "Show Applications" menus.  But I couldn't get back to the regular screen, which meant I wasn't able to use any of my applications.  I usually use the X11 window manager, but I tried it with Wayland as well and had the same experience.

Once I turned off the vertical overview extension, it was back to working normally, so I think it's the extension (interacting with something from the update I downloaded this morning) that's causing it to malfunction.

**Comment by benscruton** [comment]
I may have found the error here -- when I logged out and logged back in with Gnome Classic, the Vertical Overview displayed the following error message:

`Main.overview._overview is undefined`

Gnome Classic is pretty different, so it's possible that this is unrelated to the main issue.  But given that it stems from the activities overview and the error also mentions an overview, it seems like it may be the same thing.

**Comment by RensAlthuis** [comment]
Was there trace after that error message? 
This is working for others, so my best guess is that there is a conflict with another extension. Do you have any enabled?

**Comment by eeshugerman** [comment]
I think this is a dup of https://github.com/RensAlthuis/vertical-overview/issues/64

**Comment by benscruton** [comment]
There wasn't a trace that I was able to see, unfortunately, just the one line.  The only other extension I have enabled is "Background Logo," which is one of the defaults that comes with Fedora.

I looked over #64 and wondered if it was the same issue, but I didn't see any mention of Main.overview._overview in the description there.  I'm also on GNOME 40.4.0, whereas 64 specified it was 40.4-1 causing issues.  If I wanted to try that solution, though, would I just enter the line `this._skipTaskbarSignals = new Map();` manually into the file on my system?  Or maybe try running the terminal command mentioned [here](https://github.com/RensAlthuis/vertical-overview/issues/64#issuecomment-908786806)?
